lib/AST/ProtocolConformance.cpp

```diff
@@ -1112,30 +1112,62 @@ bool NominalTypeDecl::lookupConformance(
            conformances);
 }
 
-SmallVector<ProtocolDecl *, 2>
+bool NominalTypeDecl::lookupConformanceIgnoringErrors(
+    ProtocolDecl *protocol,
+    SmallVectorImpl<ProtocolConformance *> &conformances) const {
+  auto resultOrError = lookupConformance(protocol, conformances);
+  assert(resultOrError);
+  if (!resultOrError)
+    llvm::consumeError(resultOrError.takeError());
```

Comment:
Should we make all of these checks dropping the errors specific to the debugger and keep them in release builds? Most of our qualifications uses release builds and the changes here are pervasive enough that it could make us miss some regression.

On that idea, maybe we don't need the `IgnoringErrors` variants if we can change the existing services to silently drop the errors only for the debugger.